### PR TITLE
Adding command support to enable and disable Org 2Fa command

### PR DESCRIPTION
**Pull request by tahmidefaz**

### CODE REVIEW

File: src/api/PostManage2fa.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    'Authorization': `Bearer ${token}`
+  }
+
+  const config: AxiosRequestConfig = {
+    headers: headers
+  }
+
+  const reqBody = {"authenticationFactors":{"otp":{"required": request.enable_org_2fa}}}
+
+  return axiosInstance.post('https://sso.stage.redhat.com/auth/realms/redhat-external/apis/organizations/v1/my/authentication-policy', reqBody, config);
```
Comment (tahmidefaz) [inline src/api/PostManage2fa.ts:24]: Is this accurate?

File: src/api/PostManage2fa.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { AxiosRequestConfig } from 'axios';
+import axiosInstance from '@redhat-cloud-services/frontend-components-utilities/interceptors/interceptors';
+import useChrome from '@redhat-cloud-services/frontend-components/useChrome';
+
+export interface PostManageOrg2faRequest {
+  enable_org_2fa: boolean;
+}
+
+export const postManageOrg2fa = async (request: PostManageOrg2faRequest) => {
+  const chrome = useChrome()
```
Comment (josejulio) [inline src/api/PostManage2fa.ts:10]: We can't call `useChrome` in here - check how we pass the options in the command processor and maybe you could pass the `auth` object.

File: src/api/PostManage2fa.ts (hunk 1 above)
Comment (josejulio) [inline src/api/PostManage2fa.ts:24]: We might need to configure or compute this based on the domain so it matches prod/stage

**Comment by josejulio** [comment]
Try running `npm run lint:js:fix` to fix most of the lint errors - it should spit a shorter list for the ones that need to be fixed manually

**Review by josejulio — APPROVED** [review]

**Pull request merged**